test/e2e/steps/hooks.js

```diff
@@ -10,9 +11,23 @@ Before('@loginHook', function () {
   LoginPage.submitButton.click();
 });
 
+After('@logoutHook', function () {
+  LoginPage.profileButton.click();
+  LoginPage.logoutLink.click();
+});
+
+Before('@createList', function () {
+  DashboardPage.addListButton.moveTo();
+  DashboardPage.addListButton.click();
+  DashboardPage.newListButton.click();
+  DashboardPage.setNewListInput(Utils.generateID());
+  DashboardPage.boardTab.click();
+});
+
 After('@deleteList', function () {
+  DashboardPage.listSettingsButton.moveTo();
   DashboardPage.listSettingsButton.click();
   DashboardPage.deleteButton.click();
-  DashboardPage.confirmDeleteButton.waitForExist(3000);
+  browser.pause(1000);
```

Comment:
it's really necessary this line?? would be better if we find another way to wait for an element